src/server/editorServices.ts

```diff
@@ -360,7 +339,7 @@ namespace ts.server {
         /**
          * projects specified by a tsconfig.json file

```

Comment:
Would be useful to specify what the key is supposed to be. Same for `mapOfConfigFilePresense` and `pendingProjectUpdates`.